resources/faq.md

```diff
@@ -19,52 +19,14 @@ Meilisearch is really **easy to use** and thus accessible to all kinds of develo
 
 We also provide a lot of tools, including [SDKs](/learn/what_is_meilisearch/sdks.md), to help you integrate easily Meilisearch in your project. We're adding new tools every day!
 
-Plus, you can [contact us](#how-can-i-contact-the-meilisearch-team) if you need any help. We will answer for sure!
-
-## Do I need to configure Meilisearch to get it working?
-
-Meilisearch configuration works out-of-the-box. It means, by default, Meilisearch configures necessary settings for providing a powerful and relevant search.
-
-For example, without requiring any configuration, Meilisearch is typo tolerant.
-Type `craete an index` in the search bar of this documentation to experience the typo tolerance of our search engine.
-
-To find out more about the relevancy of Meilisearch, take a look at this detailed [explanation](/learn/core_concepts/relevancy.md#ranking-rules).
-
-However, Meilisearch is of course [highly customizable](/reference/api/settings.md) in order to adapt the search to your needs by setting synonyms, stop words, and custom ranking rules.
+Plus, you can [contact us](/learn/what_is_meilisearch/contact.md) if you need any help. We will answer for sure!
```

Comment:
Am I a terrible person for reading the last sentence ("we will answer for sure!") as extremely sarcastic? Perhaps it's best to remove it?